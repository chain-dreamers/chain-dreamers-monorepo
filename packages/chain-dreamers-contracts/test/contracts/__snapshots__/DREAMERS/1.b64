data:application/json;base64,ZGF0YTphcHBsaWNhdGlvbi9qc29uLHsiaW1hZ2VfZGF0YSI6ICIlM2NzdmclMjB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnJTIwdmlld0JveD0nMCUyMDAlMjAyNTUlMjAyNTUnJTIwd2lkdGg9JzUwMHB4JyUyMGhlaWdodD0nNTAwcHgnJTNlJTNjcGF0aCUyMGQ9J00wLDBIMjU1VjI1NUgwVjAnJTIwZmlsbD0nJTIzQURCNTQ4JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEzNywxNDVDNiwxMzMsMCwyMSwwLDIxVjgwQzAsODAsNiwxOTIsMTM3LDIwNEMyMjgsMjEyLDI1NSwyNTUsMjU1LDI1NVYxOTZDMjU1LDE5NiwyMjgsMTUzLDEzNywxNDUnJTIwZmlsbD0nJTIzRjVGRTcxJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEzNywyMUMxMDcsMTgsODQsMTEsNjYsMEgzN0M1NywyMSw4OCwzOSwxMzcsNDRDMjI4LDUyLDI1NSw5NSwyNTUsOTVWNzJDMjU1LDcyLDIyOCwyOSwxMzcsMjEnJTIwZmlsbD0nJTIzRjVGRTcxJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE3MywyMTdIODJDNTQsMjI0LDQ4LDI1NSw0OCwyNTVIMjA3QzIwNywyNTUsMjAxLDIyNCwxNzMsMjE3JyUyMGZpbGw9JyUyMzVDNzk0MCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MiwyMTFIMTczVjI1NUg4MlYyMTEnJTIwZmlsbD0nJTIzREZENUExJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTgzLDIxMVYyMjVDOTYsMjM4LDExMSwyNDYsMTI3LDI0NkMxNDQsMjQ2LDE1OSwyMzgsMTcyLDIyNVYyMTFIODMnJTIwZmlsbD0nJTIzQ0VDMDdEJy8lM2UlM2NwYXRoJTIwZD0nTTE5NywxMDZINThDNDYsMTA2LDM2LDExNywzNiwxMzFDMzYsMTQ1LDQ2LDE1Niw1OCwxNTZIMTk3QzIwOSwxNTYsMjE5LDE0NSwyMTksMTMxQzIxOSwxMTcsMjA5LDEwNiwxOTcsMTA2JyUyMGZpbGw9JyUyM0E4QTA3NyclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OCwxMzVBNjksOTksMCwxLDAsMTk3LDEzNUE2OSw5OSwwLDEsMCw1OCwxMzUnJTIwZmlsbD0nJTIzREZENUExJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5NiwxMjBDMTk2LDEyMCwyMDksMTE4LDIxMCwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OSwxMjBDNTksMTIwLDQ2LDExOCw0NSwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J003OSwxMzVDNzksODYsMTA0LDQ1LDEzNiwzOEE0NCw0NCwwLDAsMCwxMjYsMzdDODksMzcsNTksODEsNTksMTM1QzU5LDE5MCw4OSwyMzMsMTI2LDIzM0E1MCw1MCwwLDAsMCwxMzYsMjMyQzEwNCwyMjUsNzksMTg1LDc5LDEzNSclMjBmaWxsPSclMjNDRUMwN0QnLyUzZSUzY3BhdGglMjBkPSdNMTYyLDE4MEMxNjIsMTgwLDE1OCwxODAsMTU1LDE4NUE1LDUsMCwwLDEsMTUwLDE4OFYxODhBNSw1LDAsMCwxLDE0NiwxODBWMTgwQzE0NiwxODAsMTQwLDE4MCwxMzgsMTg0QzEzNiwxODksMTMxLDE4OCwxMjcsMTg4QzEyNCwxODgsMTE5LDE4OSwxMTcsMTg0QzExNSwxODAsMTA5LDE4MCwxMDksMTgwVjE4MEE1LDUsMCwwLDEsMTA1LDE4OFYxODhBNSw1LDAsMCwxLDEwMCwxODVDOTcsMTgwLDkzLDE4MCw5MywxODBDOTMsMTgwLDkwLDE5NiwxMjcsMTk2QzE2NSwxOTYsMTYyLDE4MCwxNjIsMTgwJyUyMGZpbGw9JyUyMzNFMkYxRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNDQsMTk1SDExMUE1LDUsMCwwLDAsMTExLDIwNkgxNDRBNSw1LDAsMSwwLDE0NCwxOTUnJTIwZmlsbD0nJTIzMDAwMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE0NCwxOTVIMTExQTUsNSwwLDAsMCwxMDYsMTk5SDE0OUE1LDUsMCwwLDAsMTQ0LDE5NSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTExLDIwNkgxNDRBNSw1LDAsMCwwLDE0OCwyMDNIMTA3QTUsNSwwLDAsMCwxMTEsMjA2JyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMTgsMTgySDEzNkMxNDMsMTgyLDE1MSwxNzUsMTM2LDE1N0MxMzYsMTQ3LDEyNywxNDUsMTI3LDE0NSclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEyNywxNjlBNCw0LDAsMCwwLDEyNywxNzdBNCw0LDAsMCwwLDEyNywxNjknJTIwZmlsbD0nJTIzRkZGRjAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEyNywxNjlBMywzLDAsMCwwLDEyNywxNzZBMywzLDAsMCwxLDEyNiwxNzRBMywzLDAsMCwxLDEzMCwxNzBWMTcwQTMsMywwLDAsMCwxMjcsMTY5JyUyMGZpbGw9JyUyM0FGQUEwQycvJTNlJTNjcGF0aCUyMGQ9J005OCw4N0MxMTIsODcsMTA5LDk4LDEyNCw5NUMxNDAsOTMsMTQ1LDgxLDE1Nyw5MSclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTExMCwxMDFDMTE4LDEwMSwxMTYsMTA3LDEyNiwxMDZDMTM1LDEwNCwxMzgsOTcsMTQ1LDEwMyclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk4LDE1MUM5MSwxNDksODQsMTQzLDgwLDEyOEM4MCwxMjgsNjYsMTU2LDkzLDE2NUE3LDcsMCwwLDAsMTAyLDE2MUE3LDcsMCwwLDAsOTgsMTUxJyUyMGZpbGw9JyUyM0E2QTZBNiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MywxMTdDOTgsMTEzLDEwNywxMjEsMTExLDEyNUE2LDYsMCwwLDAsMTE3LDEyOFYxMjhBNiw2LDAsMCwwLDEyMSwxMTdDMTAxLDk2LDgzLDExNyw4MywxMTcnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEzNCwxMTdBNiw2LDAsMCwwLDEzOCwxMjhWMTI4QTYsNiwwLDAsMCwxNDQsMTI1QzE0OCwxMjEsMTU3LDExMywxNzIsMTE3QzE3MiwxMTcsMTU0LDk2LDEzNCwxMTcnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwNiwxMjJBMTQsMTQsMCwwLDAsMTAxLDEyMkExNSwxNSwwLDAsMCwxMDEsMTUyQTE0LDE0LDAsMCwwLDEwNiwxNTFBMTUsMTUsMCwwLDEsMTA2LDEyMiclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTQ4LDEzN0ExNSwxNSwwLDAsMSwxNTgsMTIyQTE0LDE0LDAsMCwwLDE1NCwxMjJBMTUsMTUsMCwwLDAsMTU0LDE1MkExNCwxNCwwLDAsMCwxNTgsMTUxQTE1LDE1LDAsMCwxLDE0OCwxMzcnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk2LDEzN0ExMCwxNCwwLDEsMCwxMTcsMTM3QTEwLDE0LDAsMSwwLDk2LDEzNyclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTQ4LDEzN0ExMCwxNCwwLDEsMCwxNjksMTM3QTEwLDE0LDAsMSwwLDE0OCwxMzcnJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwMSwxMzZBNSw1LDAsMSwwLDExMSwxMzZBNSw1LDAsMSwwLDEwMSwxMzYnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE1MywxMzZBNSw1LDAsMSwwLDE2NCwxMzZBNSw1LDAsMSwwLDE1MywxMzYnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTQ0LDEwNkE2LDYsMCwxLDAsNTAsMTEyQTYsNiwwLDAsMCw0NCwxMDYnJTIwZmlsbD0nJTIzODQ0MUE4JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTQ2LDEwN0g0N0E2LDYsMCwwLDAsNDMsMTA3QTUsNSwwLDAsMCw0MywxMThBNSw1LDAsMCwxLDQ2LDEwN1YxMDcnJTIwZmlsbD0nJTIzNUQwNThDJy8lM2UlM2NwYXRoJTIwZD0nTTM2LDEyOUE2LDYsMCwxLDAsNDIsMTM1QTYsNiwwLDAsMCwzNiwxMjknJTIwZmlsbD0nJTIzNTE4REQzJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTM4LDEzMUgzOUE1LDUsMCwwLDAsMzYsMTMwQTUsNSwwLDAsMCwzMCwxMzVBNSw1LDAsMCwwLDM1LDE0MUE1LDUsMCwwLDEsMzMsMTM2QTUsNSwwLDAsMSwzOCwxMzEnJTIwZmlsbD0nJTIzMDA0RUJFJy8lM2UlM2NwYXRoJTIwZD0nTTM5LDE1MkE2LDYsMCwxLDAsNTEsMTUyQTYsNiwwLDEsMCwzOSwxNTInJTIwZmlsbD0nJTIzQUZEMzYxJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTQ4LDE0N1YxNDdBNSw1LDAsMCwwLDQ1LDE0NkE1LDUsMCwwLDAsMzksMTUyQTUsNSwwLDAsMCw0NSwxNTdBNSw1LDAsMCwxLDQyLDE1M0E1LDUsMCwwLDEsNDgsMTQ3JyUyMGZpbGw9JyUyMzQ3NzIwMCcvJTNlJTNjcGF0aCUyMGQ9J00yMDUsMTEyQTYsNiwwLDEsMCwyMTgsMTEyQTYsNiwwLDEsMCwyMDUsMTEyJyUyMGZpbGw9JyUyMzg0NDFBOCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00yMTIsMTA3QTUsNSwwLDAsMCwyMDgsMTA3VjEwN0E1LDUsMCwwLDEsMjE0LDExM0E1LDUsMCwwLDEsMjEyLDExOEE1LDUsMCwwLDAsMjEyLDEwN1YxMDcnJTIwZmlsbD0nJTIzNUQwNThDJy8lM2UlM2NwYXRoJTIwZD0nTTIxOSwxMjlBNiw2LDAsMSwwLDIyNiwxMzVBNiw2LDAsMCwwLDIxOSwxMjknJTIwZmlsbD0nJTIzNTE4REQzJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTIxOSwxMzBBNSw1LDAsMCwwLDIxNiwxMzFWMTMxQTUsNSwwLDAsMSwyMTksMTQxQTUsNSwwLDAsMCwyMTksMTMwVjEzMCclMjBmaWxsPSclMjMwMDRFQkUnLyUzZSUzY3BhdGglMjBkPSdNMjEwLDE0NkE2LDYsMCwxLDAsMjE2LDE1MkE2LDYsMCwwLDAsMjEwLDE0NiclMjBmaWxsPSclMjNBRkQzNjEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMjEwLDE0NkE2LDYsMCwwLDAsMjA3LDE0N1YxNDdBNSw1LDAsMCwxLDIxMywxNTNBNSw1LDAsMCwxLDIxMCwxNTdBNSw1LDAsMCwwLDIxNiwxNTJBNSw1LDAsMCwwLDIxMCwxNDYnJTIwZmlsbD0nJTIzNDc3MjAwJy8lM2UlM2NwYXRoJTIwZD0nTTE2Miw0NEMxNjAsMzQsMTA4LDE4LDgzLDQ0QzcxLDQ1LDU2LDUyLDYwLDY4QzYwLDY4LDQ3LDEwMCw2NywxMjdDNjcsMTI3LDgxLDExMyw3Niw5NUM4Niw5Nyw4OCw5MCw4NSw4NUM4NSw4NSw5Miw5MCwxMTIsODVDMTEzLDk4LDE1MCwxMzAsMTk5LDExNkMxOTksMTE2LDIwNSw2MiwxNjIsNDQnJTIwZmlsbD0nJTIzNEIzRjNDJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE0NSwzNEE4Miw4MiwwLDAsMCwxMjIsMzBDMTEwLDMwLDk1LDMzLDgzLDQ1VjQ1Qzc0LDQ1LDY3LDQ5LDYzLDU0QzU5LDU5LDU5LDYzLDYwLDY4VjY4QzYwLDY5LDQ3LDEwMCw2NywxMjdBNDMsNDMsMCwwLDAsNzIsMTE5QzYyLDk3LDcxLDc0LDcyLDcyQzcwLDY2LDcxLDYxLDc0LDU3Qzc4LDUyLDg2LDQ4LDk1LDQ3QzEwNywzNiwxMjMsMzMsMTM0LDMzQTgwLDgwLDAsMCwxLDE0NSwzNCclMjBmaWxsPSclMjMyODFFMUMnLyUzZSUzY3BhdGglMjBkPSdNMTI0LDg5QTcyLDcyLDAsMCwxLDExNSw5MUMxMjEsMTAwLDEzNCwxMTAsMTUxLDExNkMxMzUsMTA4LDEyNSw5NiwxMjQsODknJTIwZmlsbD0nJTIzMjgxRTFDJy8lM2UlM2NwYXRoJTIwZD0nTTE4Miw2MUE2LDYsMCwwLDAsMTczLDYxTDEyMSwxMTNDMTE0LDEwNSwxMDAsMTAzLDkwLDExMEg1OEM1MiwxMTAsNTIsMTIwLDU4LDEyMEg4MkM3NywxMjksNzksMTQxLDg3LDE0OEw2MCwxNzVDNTYsMTc5LDU5LDE4Niw2NSwxODVBNiw2LDAsMCwwLDcwLDE4NEw5OCwxNTVDMTE2LDE1OSwxMzMsMTQyLDEyOCwxMjRMMTgyLDcwQTYsNiwwLDAsMCwxODIsNjEnJTIwZmlsbD0nJTIzMTgxODE4JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk0LDEzMUEyNCwyNCwwLDAsMSwxMTEsMTA3QTI2LDI2LDAsMCwwLDEwNCwxMDZBMjQsMjQsMCwwLDAsMTA0LDE1NUEyNiwyNiwwLDAsMCwxMTEsMTU0QTI0LDI0LDAsMCwxLDk0LDEzMSclMjBmaWxsPSclMjMwMDAwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTQxLDE5NUEzLDMsMCwwLDAsMTM2LDE5NkEzLDMsMCwwLDAsMTM2LDIwMEwxNDIsMjA2TDE0NywyMDFMMTQxLDE5NSclMjBmaWxsPSclMjNEMjc2M0EnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTQ3LDIwMUwxNDIsMjA2TDE4MSwyNDVMMTg1LDI0MEwxNDcsMjAxVjIwMSclMjBmaWxsPSclMjNFQUVBRUEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTgxLDIzNUwxNzYsMjQwTDE4MSwyNDVMMTg1LDI0MEwxODEsMjM1VjIzNSclMjBmaWxsPSclMjNDQzM0MkEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTgzLDIwMUExMywxMywwLDEsMCwxOTcsMjE0QTEzLDEzLDAsMCwwLDE4MywyMDEnJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5OCwxNzZBMTcsMTcsMCwxLDAsMjE2LDE5NEExNywxNywwLDAsMCwxOTgsMTc2JyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xODUsMTc2QTEyLDEyLDAsMSwwLDE5NywxODhBMTIsMTIsMCwwLDAsMTg1LDE3NiclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTkzLDE0OUE4LDgsMCwxLDAsMjAyLDE1OEE4LDgsMCwwLDAsMTkzLDE0OSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTg1LDE3MEE2LDYsMCwwLDAsMTg1LDE4MkE2LDYsMCwxLDAsMTg1LDE3MCclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTc1LDIyOUE3LDcsMCwxLDAsMTkwLDIyOUE3LDcsMCwxLDAsMTc1LDIyOSclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTg3LDE0NEE1LDUsMCwxLDAsMTg3LDEzM0E1LDUsMCwwLDAsMTg3LDE0NCclMjBmaWxsPSclMjNGRkZGRkYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3N0eWxlJTNlcGF0aHtzdHJva2Utd2lkdGg6MC43MX0lM2Mvc3R5bGUlM2UlM2Mvc3ZnJTNlIiwgIm5hbWUiOiAiRHJlYW1lciUyMCUyMzEiLCAiZGVzY3JpcHRpb24iOiAiUnVubmVycyUyMHJ1biwlMjBidXQlMjBzb21ldGltZXMlMjB0aGV5JTIwZHJlYW0uJTIwVGhpcyUyMGlzJTIwb25lJTIwb2YlMjB0aGVpciUyMGRyZWFtcy4iLCJhdHRyaWJ1dGVzIjogIiJ9