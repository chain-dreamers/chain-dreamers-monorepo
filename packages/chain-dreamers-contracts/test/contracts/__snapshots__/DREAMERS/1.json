data:application/json,{"image_data": "%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0,0H255V255H0V0'%20fill='%23ADB548'%20stroke='%23000'/%3e%3cpath%20d='M137,145C6,133,0,21,0,21V80C0,80,6,192,137,204C228,212,255,255,255,255V196C255,196,228,153,137,145'%20fill='%23F5FE71'%20stroke='%23000'/%3e%3cpath%20d='M137,21C107,18,84,11,66,0H37C57,21,88,39,137,44C228,52,255,95,255,95V72C255,72,228,29,137,21'%20fill='%23F5FE71'%20stroke='%23000'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%235C7940'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23DFD5A1'%20stroke='%23000'/%3e%3cpath%20d='M83,211V225C96,238,111,246,127,246C144,246,159,238,172,225V211H83'%20fill='%23CEC07D'/%3e%3cpath%20d='M197,106H58C46,106,36,117,36,131C36,145,46,156,58,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23A8A077'%20stroke='%23000'/%3e%3cpath%20d='M58,135A69,99,0,1,0,197,135A69,99,0,1,0,58,135'%20fill='%23DFD5A1'%20stroke='%23000'/%3e%3cpath%20d='M196,120C196,120,209,118,210,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,120C59,120,46,118,45,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M79,135C79,86,104,45,136,38A44,44,0,0,0,126,37C89,37,59,81,59,135C59,190,89,233,126,233A50,50,0,0,0,136,232C104,225,79,185,79,135'%20fill='%23CEC07D'/%3e%3cpath%20d='M162,180C162,180,158,180,155,185A5,5,0,0,1,150,188V188A5,5,0,0,1,146,180V180C146,180,140,180,138,184C136,189,131,188,127,188C124,188,119,189,117,184C115,180,109,180,109,180V180A5,5,0,0,1,105,188V188A5,5,0,0,1,100,185C97,180,93,180,93,180C93,180,90,196,127,196C165,196,162,180,162,180'%20fill='%233E2F1F'%20stroke='%23000'/%3e%3cpath%20d='M144,195H111A5,5,0,0,0,111,206H144A5,5,0,1,0,144,195'%20fill='%23000000'/%3e%3cpath%20d='M144,195H111A5,5,0,0,0,106,199H149A5,5,0,0,0,144,195'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M111,206H144A5,5,0,0,0,148,203H107A5,5,0,0,0,111,206'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M118,182H136C143,182,151,175,136,157C136,147,127,145,127,145'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M127,169A4,4,0,0,0,127,177A4,4,0,0,0,127,169'%20fill='%23FFFF00'%20stroke='%23000'/%3e%3cpath%20d='M127,169A3,3,0,0,0,127,176A3,3,0,0,1,126,174A3,3,0,0,1,130,170V170A3,3,0,0,0,127,169'%20fill='%23AFAA0C'/%3e%3cpath%20d='M98,87C112,87,109,98,124,95C140,93,145,81,157,91'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M110,101C118,101,116,107,126,106C135,104,138,97,145,103'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M98,151C91,149,84,143,80,128C80,128,66,156,93,165A7,7,0,0,0,102,161A7,7,0,0,0,98,151'%20fill='%23A6A6A6'%20stroke='%23000'/%3e%3cpath%20d='M83,117C98,113,107,121,111,125A6,6,0,0,0,117,128V128A6,6,0,0,0,121,117C101,96,83,117,83,117'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M134,117A6,6,0,0,0,138,128V128A6,6,0,0,0,144,125C148,121,157,113,172,117C172,117,154,96,134,117'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M106,122A14,14,0,0,0,101,122A15,15,0,0,0,101,152A14,14,0,0,0,106,151A15,15,0,0,1,106,122'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M148,137A15,15,0,0,1,158,122A14,14,0,0,0,154,122A15,15,0,0,0,154,152A14,14,0,0,0,158,151A15,15,0,0,1,148,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M96,137A10,14,0,1,0,117,137A10,14,0,1,0,96,137'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M148,137A10,14,0,1,0,169,137A10,14,0,1,0,148,137'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M101,136A5,5,0,1,0,111,136A5,5,0,1,0,101,136'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M153,136A5,5,0,1,0,164,136A5,5,0,1,0,153,136'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M44,106A6,6,0,1,0,50,112A6,6,0,0,0,44,106'%20fill='%238441A8'%20stroke='%23000'/%3e%3cpath%20d='M46,107H47A6,6,0,0,0,43,107A5,5,0,0,0,43,118A5,5,0,0,1,46,107V107'%20fill='%235D058C'/%3e%3cpath%20d='M36,129A6,6,0,1,0,42,135A6,6,0,0,0,36,129'%20fill='%23518DD3'%20stroke='%23000'/%3e%3cpath%20d='M38,131H39A5,5,0,0,0,36,130A5,5,0,0,0,30,135A5,5,0,0,0,35,141A5,5,0,0,1,33,136A5,5,0,0,1,38,131'%20fill='%23004EBE'/%3e%3cpath%20d='M39,152A6,6,0,1,0,51,152A6,6,0,1,0,39,152'%20fill='%23AFD361'%20stroke='%23000'/%3e%3cpath%20d='M48,147V147A5,5,0,0,0,45,146A5,5,0,0,0,39,152A5,5,0,0,0,45,157A5,5,0,0,1,42,153A5,5,0,0,1,48,147'%20fill='%23477200'/%3e%3cpath%20d='M205,112A6,6,0,1,0,218,112A6,6,0,1,0,205,112'%20fill='%238441A8'%20stroke='%23000'/%3e%3cpath%20d='M212,107A5,5,0,0,0,208,107V107A5,5,0,0,1,214,113A5,5,0,0,1,212,118A5,5,0,0,0,212,107V107'%20fill='%235D058C'/%3e%3cpath%20d='M219,129A6,6,0,1,0,226,135A6,6,0,0,0,219,129'%20fill='%23518DD3'%20stroke='%23000'/%3e%3cpath%20d='M219,130A5,5,0,0,0,216,131V131A5,5,0,0,1,219,141A5,5,0,0,0,219,130V130'%20fill='%23004EBE'/%3e%3cpath%20d='M210,146A6,6,0,1,0,216,152A6,6,0,0,0,210,146'%20fill='%23AFD361'%20stroke='%23000'/%3e%3cpath%20d='M210,146A6,6,0,0,0,207,147V147A5,5,0,0,1,213,153A5,5,0,0,1,210,157A5,5,0,0,0,216,152A5,5,0,0,0,210,146'%20fill='%23477200'/%3e%3cpath%20d='M162,44C160,34,108,18,83,44C71,45,56,52,60,68C60,68,47,100,67,127C67,127,81,113,76,95C86,97,88,90,85,85C85,85,92,90,112,85C113,98,150,130,199,116C199,116,205,62,162,44'%20fill='%234B3F3C'%20stroke='%23000'/%3e%3cpath%20d='M145,34A82,82,0,0,0,122,30C110,30,95,33,83,45V45C74,45,67,49,63,54C59,59,59,63,60,68V68C60,69,47,100,67,127A43,43,0,0,0,72,119C62,97,71,74,72,72C70,66,71,61,74,57C78,52,86,48,95,47C107,36,123,33,134,33A80,80,0,0,1,145,34'%20fill='%23281E1C'/%3e%3cpath%20d='M124,89A72,72,0,0,1,115,91C121,100,134,110,151,116C135,108,125,96,124,89'%20fill='%23281E1C'/%3e%3cpath%20d='M182,61A6,6,0,0,0,173,61L121,113C114,105,100,103,90,110H58C52,110,52,120,58,120H82C77,129,79,141,87,148L60,175C56,179,59,186,65,185A6,6,0,0,0,70,184L98,155C116,159,133,142,128,124L182,70A6,6,0,0,0,182,61'%20fill='%23181818'%20stroke='%23000'/%3e%3cpath%20d='M94,131A24,24,0,0,1,111,107A26,26,0,0,0,104,106A24,24,0,0,0,104,155A26,26,0,0,0,111,154A24,24,0,0,1,94,131'%20fill='%23000000'/%3e%3cpath%20d='M141,195A3,3,0,0,0,136,196A3,3,0,0,0,136,200L142,206L147,201L141,195'%20fill='%23D2763A'%20stroke='%23000'/%3e%3cpath%20d='M147,201L142,206L181,245L185,240L147,201V201'%20fill='%23EAEAEA'%20stroke='%23000'/%3e%3cpath%20d='M181,235L176,240L181,245L185,240L181,235V235'%20fill='%23CC342A'%20stroke='%23000'/%3e%3cpath%20d='M183,201A13,13,0,1,0,197,214A13,13,0,0,0,183,201'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M198,176A17,17,0,1,0,216,194A17,17,0,0,0,198,176'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M185,176A12,12,0,1,0,197,188A12,12,0,0,0,185,176'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M193,149A8,8,0,1,0,202,158A8,8,0,0,0,193,149'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M185,170A6,6,0,0,0,185,182A6,6,0,1,0,185,170'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M175,229A7,7,0,1,0,190,229A7,7,0,1,0,175,229'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M187,144A5,5,0,1,0,187,133A5,5,0,0,0,187,144'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e", "name": "Dreamer%20%231", "description": "Runners%20run,%20but%20sometimes%20they%20dream.%20This%20is%20one%20of%20their%20dreams.","attributes": ""}